pkg/unikontainers/config.go

```diff
@@ -60,6 +60,19 @@ type UnikernelConfig struct {
 	MountRootfs      string `json:"com.urunc.unikernel.mountRootfs"`
 }
 
+func (c *UnikernelConfig) validate() error {
+	if c.UnikernelType == "" {
+		return errors.New("unikernel configuration is missing mandatory field: com.urunc.unikernel.unikernelType")
+	}
+	if c.Hypervisor == "" {
+		return errors.New("unikernel configuration is missing mandatory field: com.urunc.unikernel.hypervisor")
+	}
+	if c.UnikernelBinary == "" {
+		return errors.New("unikernel configuration is missing mandatory field: com.urunc.unikernel.binary")
```

Comment:
It would be better to use fmt.Errorf here instead of errors.New. Also, it would be better to use the `c.UnikernelBinary` instead of the string `com.urunc.unikernel.unikernelBinary`.